app-admin/pinky-bar/pinky-bar-9999.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://gitlab.com/void0/pinky-bar.git"
+
+inherit git-r3
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://gitlab.com/void0/pinky-bar"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="x11 alsa +net libnl +pci dvd sensors ncurses +colours weather mpd drivetemp smartemp"
+
+DEPEND="
+	sys-devel/m4
+	sys-apps/gawk
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
```

Comment:
in Gentoo you don't directly depend on the autotools via `DEPEND`, instead you `inherit autotools` which sets these correctly.